ansible-tests/validations/network_environment.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: undercloud
+  become: false
+  vars:
+    metadata:
+      name: Validate the Heat environment file for network configuration
+      description: >
+        This validates the network environment and nic-config files
+        that specify the overcloud network configuration.
+    network_environment_path: /home/stack/network-environment.yaml
```

Comment:
Yeah, the deployers are expected to write it themselves:

http://tripleo.org/advanced_deployment/network_isolation.html

Here's a sample:

https://github.com/openstack/tripleo-heat-templates/blob/master/environments/network-environment.yaml